content/docs/guides/securing-tcp.mdx

```diff
@@ -21,154 +21,150 @@ description: This guide covers how to use Pomerium to protect TCP services such
 
-- Create a standard Pomerium configuration for your [identity provider (IdP)](/docs/identity-providers)
-- `pomerium-cli` runs on your workstation, listening on loopback for TCP connections
-- When an inbound connection is made, `pomerium-cli` proxies the connection through `pomerium`, authenticating the user if needed
-- Pomerium authorizes the connection and forwards it to the upstream service
-- The connecting application functions as normal
+Authentication and authorization configuration is shared with standard HTTP routes, and the underlying transport is still encrypted between the end-user and Pomerium.
 
-## Pre-requisites
+:::note
 
-This recipe is designed to run on a local docker-compose instance. The included configuration can be adopted for any TCP service, however.
+- Pomerium authorizes TCP connections on a **per-connection basis** (unlike HTTP requests, which it authorizes on a per-request basis)
+- Pomerium only authorizes the TCP _connection_; it does not interact with application level authorization systems
 
-- docker
-- docker-compose
-- A copy of the [example repo](https://github.com/pomerium/pomerium/tree/main/examples/tcp/) checked out
-- Valid credentials for your OIDC provider
-- The [Pomerium Client](/docs/deploy/clients/pomerium-cli) installed
-- (Optional) `mkcert` to generate locally trusted certificates
+:::
 
-## Certificates (optional)
+## Overview
 
-This demo comes with its own certificates, but `pomerium-cli` and your browser will not trust them by default. You may instead provide your own or use [mkcert](https://github.com/FiloSottile/mkcert) to generate locally trusted certificates.
+Review the steps below to understand how you will secure TCP connections with Pomerium:
 
-After installing `mkcert`, run the following inside the example repo:
-
-```bash
-mkcert -install
-   mkcert '*.localhost.pomerium.io'
-```
+1. Run Pomerium CLI's `tcp` command in your workstation to listen for TCP connections on loopback
+1. When an inbound connection is made, Pomerium CLI proxies the connection through `pomerium`, authenticating the user if needed
+1. Pomerium authorizes the connection and forwards it to the upstream service
 
-This will install a trusted CA and generate a new wildcard certificate:
+## Before you start
 
-- `_wildcard.localhost.pomerium.io.pem`
-- `_wildcard.localhost.pomerium.io-key.pem`
+To complete this guide, you need:
 
-To provide your own certificates through another mechanism, please overwrite these files or update `docker-compose.yaml` accordingly.
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [Pomerium CLI](/docs/deploy/clients/pomerium-cli)
 
 ## Configure
 
 ### Pomerium
 
-Update `config.yaml` with your IdP settings and desired policy if adopting for your environment
+Add the configuration below to your configuration file:
```

Comment:
I think we should also mention that they'll need to change the domain in each policy from "example.com" to something that matches their email address.